### PR TITLE
Drop support of Python 2.7 and 3.4 ; add support of Python 3.8

### PR DESCRIPTION
**Pull request by dbaty**
Separate commits in case we want to keep supporting Python 2.7 because of... nostalgia?

Right now building other pull requests fail (and thus they cannot be merged) because some development dependencies dropped the support of Python 3.4. Example: https://travis-ci.org/github/Polyconseil/getconf/jobs/667171560.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -40,8 +40,9 @@ respecting the standard procedures in each system:
 * Use standard configuration space in ``/etc`` on traditional production servers
 * Handle environment-based configuration for cloud-based platforms
 
-``getconf`` v1.6 onwards supports 2.7, 3.3, 3.4, 3.5, 3.6 and is distributed under the two-clause BSD license.
```
Comment (rbarrois) [inline README.rst]: Small note: the way this sentence is worded gives the impression that there was a license change in version 1.10?

File: README.rst [hunk 2]
```diff
@@ -40,8 +40,9 @@ respecting the standard procedures in each system:
...
-v1.8.x will be the last versions to support 2.7 and 3.3.
+``getconf`` v1.10 onwards supports Python 3.5, 3.6, 3.7 and 3.8 and is distributed under the two-clause BSD license.
+v1.9.x are the last versions to support 2.7 and 3.4.
+v1.8.x are the last versions to support 3.3.
 The latest version of ``getconf`` to support Python 2.6 is v1.5.1.
```
Comment (rbarrois) [inline README.rst]: Is there a reason for the different wording of those "last version to support" sentences? ;)

Comment (dbaty) [reply README.rst]: I'll ask the person who wrote them first. ;) But yes, you're right, I will reformulate the last sentence.

File: README.rst (hunk 1 above)
Comment (dbaty) [reply README.rst]: Good point, I'll mention the license separately.

Comment (dbaty) [reply README.rst]: ... and while I'am at it, I'll add a `LICENSE` file for clarity.

**Pull request merged**